examples/diamond/lib.rs

```diff
@@ -0,0 +1,39 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(incomplete_features)]
+#![feature(specialization)]
```

Comment:
```suggestion
#![feature(min_specialization)]
```

Reply:
we need specialization if we want to impl DiamondCut for T and then for DiamondLoupeStorage